app/models/concerns/media.rb

```diff
@@ -1,6 +1,8 @@
 module Media
   extend ActiveSupport::Concern
 
+  STATUSES = %w[nya upcoming current finished].freeze
```

Comment:
Yeah not yet airing. I didn't know what to call it so just went with what the client is already using. Should I change it?